stl/src/format.cpp

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+// Implements a win32 API wrapper for <format>
+
+// This must be as small as possible, because its contents are
+// injected into the msvcprt.lib and msvcprtd.lib import libraries.
+// Do not include or define anything else here.
+// In particular, basic_string must not be included here.
+
+#include <xfilesystem_abi.h>
+#include <xlocinfo.h>
+
+#include <Windows.h>
+
+static_assert(__std_code_page::_Acp == __std_code_page{CP_ACP});
+
+extern "C" [[nodiscard]] __std_win_error __stdcall __std_get_cvt(

```

Comment:
come to think of it why is this `__stdcall` instead of `__cdecl` (which is more standard). I don't really care, but I'd think cdecl would be more standard than stdcall

Reply:
There are far more `__cdecl` ABIs in the library, but we've been preferring `__stdcall` for signatures without varargs since 2016 or so. Billy made a convincing argument that `__stdcall` reduces overall codesize since the callee cleans up the arguments in one place rather than callers doing so at every callsite.

Reply:
I suppose stdcall probably results in smaller code on the caller side.